src/CodiceFiscale/Subject.php

```diff
@@ -71,7 +76,7 @@ public function setName($name)
     /**
      * Get the name.
      *
-     * @return Returns the name.
+     * @return string Returns the name.
```

Comment:
You could remove Returns here and use an uppercase **T** for the initial **the**.